packages/bpk-component-graphic-promotion/src/__snapshots__/BpkGraphicPromo-test.js.snap

```diff
@@ -11,6 +11,7 @@ exports[`BpkGraphicPromo should display the tagline and no sponsor for non-spons
     <div
       aria-hidden="true"
       class="bpk-graphic-promo__container bpk-graphic-promo__container--start"
+      id="undefined__content"
```

Comment:
Wouldn't it make more sense to not pass an id here in case `contentId` is undefined?

Reply:
Oh, you are right, I missed that. Sorting it out now.

Reply:
Sorted. Thanks for spotting this! 😄